vectordatasource/transform.py

```diff
@@ -8129,7 +8129,7 @@ def min_zoom_filter(ctx):
         for feature in features:
             _, props, _ = feature
             min_zoom = props.get('min_zoom')
-            if min_zoom is not None and min_zoom <= nominal_zoom + 1:
+            if min_zoom is not None and min_zoom <= nominal_zoom + 0.5:
```

Comment:
Rather than change the zoom range we include in the tile (i.e: `N` to `N+1`), I think it would be better to change the min zoom we're applying from NE. For example, changing [the setting code](https://github.com/tilezen/vector-datasource/blob/1f74fe90aacfa9e6a8201dff94440eaeedb2b2c0/vectordatasource/transform.py#L8150-L8157) to assign `props['min_zoom'] = min_zoom + 0.5` and similar for `max_zoom`. The two methods should be mathematically identical.

Reply:
Hmm, I don't like changing the `min_zoom` values to be inconsistent with Natural Earth – mostly because its had for me to wrap my head around as the author of Natural Earth! – but also because it's it's just values in the x.6 to x.9 range that should be ceiling'd up to x+1.0 to be more consistent?

Either way the places layer should be < nominal_zoom +1 not <= to avoid the Poland example?

Reply:
Attempting fix in https://github.com/tilezen/vector-datasource/pull/1734/commits/9914c2733f102b0be2e77c09edc6b86799d8dda9.